.prettierrc

```diff
@@ -6,3 +6,4 @@ bracketSpacing: true
 parenSpacing: true
 bracketSameLine: false
 semi: true
+trailingComma: 'es5'
```

Comment:
Confirming this is necessary since the default value changed from `es5` to `all` since `v3`.